### DISCUSSION TITLE
[SOLVED] Disable SASL on SMTP to prevent password attacks

### DISCUSSION THREAD

**Discussion by plittlefield** [docker-mailserver/docker-mailserver#3071] [General]
Hello All,

My first post, so be gentle :)

I  have port 587 restricted to a single IP address for sending authenticated emails.

I am seeing a LOT of these attempted password logins through port 25 with SASL ...

```
Feb  7 16:16:06 mail postfix/smtpd[84778]: connect from unknown[185.36.81.180]
Feb  7 16:16:08 mail dovecot: auth: passwd-file(envio,185.36.81.180): unknown user (SHA1 of given password: 0ab831)
Feb  7 16:16:10 mail postfix/smtpd[84778]: warning: unknown[185.36.81.180]: SASL LOGIN authentication failed: UGFzc3dvcmQ6
```

... how do I disable the SASL auth service on port 25 please?

Obviously, I still need to keep the TLS facility for connections from genuine servers :)

Thanks,

Paully

**Comment by plittlefield** [comment]
I have answered my own question :)

Add this line to your custom config for Postfix (data/config/postfix-main.cf) ...

`smtpd_sasl_auth_enable = no`

... and restart your Docker container then hey presto, no more bots trying my Dovecot password file!

**Reply by casperklein** [reply]
This will be the default in the next major release (https://github.com/docker-mailserver/docker-mailserver/pull/3006).
